src/NuGet.Core/NuGet.ProjectModel/MacroStringsUtility.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace NuGet.ProjectModel
+{
+    // TODO NK - Use OS Aware comparer.
+    // Hide behind environment variable, and require it to always be set for the time being.
+    // Add JsonPackageReader and PackageSpecWriter tests.
+    internal static class MacroStringsUtility
+    {
+        internal static string UserProfileMacro = "$(UserProfile)";
```

Comment:
`UserProfile` is an environment variable on Windows, but not Linux (no idea about Mac).  `HOME` is an environment variable on Linux, but not Windows, but it fewer characters. So, I like Home more than UserProfile.

Reply:
Yeah, I think in order to cover all scenarios, I'll need to use both Home and UserSettingsDirectory. 

On linux/mac those are the same directory, but on windows they aren't, which is the primary scenario.

Reply:
I think UserSettings is more helpful to being with though, so I think I'll with that.